modules/cluster/variables.tf

```diff
@@ -189,6 +189,18 @@ variable "critical_addons_node_group_bottlerocket_admin_container_source" {
   description = "URI of a custom admin container image"
 }
 
+variable "critical_addons_coredns_preserve" {
+  type        = string
+  default     = true
```

Comment:
I think its safe to make this `true` by default. It reduces the number of steps required to migrate, and has no effect for users who do not want to migrate.